### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by ZaryImortal** [LaksCastro/anime-dart#3]
PODERIA ACRESCENTAR A OPÇÃO DE MARCAR A SÉRIE COMO FAVORITO, DENTRO DO EPISÓDIO, AS VEZES VEMOS UM EPISÓDIO E GOSTAMOS DO ANIME AI TEMOS QUE PROCURAR PELO NOME...
![image](https://user-images.githubusercontent.com/57652636/92151536-af2eb400-edf7-11ea-8b89-8a25c451e9fd.png)
.
ACRESCENTAR UMA MANEIRA DE MARCAR COMO VÁRIOS EPISÓDIO AO MESMO TEMPO E NÃO ENTRANDO NO EPISÓDIO E MARCANDO MANUALMENTE.
![image](https://user-images.githubusercontent.com/57652636/92151722-fc128a80-edf7-11ea-93c0-d98407728c24.png)
.
O PLAYER PODERIA TER A OPÇÃO DE AVANÇAR E VOLTAR ALGUNS SEGUNDOS (SEJA CLICANDO NA TELA OU NO PRÓPRIO PLAYER), E TAMBÉM TER COMO CLICAR NA TELA COM O PLAYER PAUSADO SEM QUE O ANIME CONTINUE ( ACONTECE QUE ALGUMAS LEGENDAS SÃO MUITO RÁPIDAS E TEMOS QUE PAUSAR MAS O PLAYER FICA NA FRENTE E SE CLICAMOS NA TELA ELE CONTINUA O ANIME) DEIXAR O FUNDO DO PLAYER TRANSPARENTE EM VEZ DE BRANCO SERIA UMA BOA TAMBÉM.
.
AGORA UMA DÚVIDA PRA QUE SERVE ISSO?
![image](https://user-images.githubusercontent.com/57652636/92152456-11d47f80-edf9-11ea-98df-2bc955e75327.png)

**Comment by LaksCastro** [comment]
Olá @ZaryImortal! Obrigado pelo feedback.

Também senti falta desses atalhos, principalmente o de "marcar vários como assistido" e o de pular/voltar 5s. 

Sobre o ícone roxo na página inicial, ele serve para marcar um episódio como assistido ou não assistido sem precisar entrar na página do episódio. Caso o ícone esteja apagado, quer dizer que aquele episódio está marcado como não assistido, caso ele esteja roxo, como está na print, quer dizer que ele já foi assistido.

No momento estou trabalhando na construção de um servidor que irá permitir a exibição dos conteúdos requisitados na [Issue#2](https://github.com/LaksCastro/anime-dart/issues/2) + a integração com o AniList e MyAnimeList. Então assim que este servidor for concluído, vou começar a implementar esses atalhos que mencionou nesta Issue. 

E de verdade, muito obrigado pelo feedback, como é somente eu trabalhando no código do projeto não é algo tão rápido de implementar, mas mesmo assim, com esses feedbacks posso manter uma "To Do List" de features a serem implementadas que farão o App ficar melhor pra todo mundo.

Obrigado, e volto a repetir, sempre que tiver algo a acrescentar, pode enviar! =D

**Comment by ZaryImortal** [comment]
Só acrescentando.
-Receber notificação de episódio novo dos animes nos favoritos.
-Mudar o fundo dos animes que você já colocou nos favoritos (algo assim) 
![image](https://user-images.githubusercontent.com/57652636/92337468-5aea3500-f080-11ea-98dd-4023d129061d.png)

**Comment by LaksCastro** [comment]
Ótima parte do menu de configurações.

Sobre o DarkMode: ele segue o tema do sistema do seu celular. Então para ativar o dark mode basta mudar o tema do seu celular (Coisa que será corrigida no menu de configurações para aceitar mudar o tema sem precisar seguir o do celular)

Sobre a parte de notificações é uma sugestão bem legal apesar de ser um pouquinho complicado de implementar.

Essa parte de atalhos e indicativos de favoritos será totalmente reformada, pois estou reconstruindo o app pra aceitar tanto animes/mangás/imagens/artes. Mas com certeza to levando em consideração o que está dizendo: manter uma forma de mostrar o estado do anime com atalhos de forma simples.

Sobre a parte de criar conta para armazenar os favoritos em nuvem não é redundante já que os favoritos serão salvos usando a conta do MyAnimeList/AniList? Como foi citado na #2 Issue?

Valeeeu, @ZaryImortal!

**Comment by ZaryImortal** [comment]
> Sobre a parte de criar conta para armazenar os favoritos em nuvem não é redundante já que os favoritos serão salvos usando a conta do MyAnimeList/AniList? Como foi citado na #2 Issue?

Vou dar um exemplo: A pessoa troca de celular ou remove o app por N motivo. Ela teria que colocar os favoritos no app manualmente. Pelo menos é assim quando o app não possui contas,  mas se vc conseguir fazer com que extraia a informação da conta do myanimelist (coisa que não vi nenhum app fazer) e coloque automaticamente oq vc está lendo nos favoritos, então realmente não precisará de conta. 

Outra coisa que percebi é que os animes não possuem descrição.

**Comment by LaksCastro** [comment]
Entendi, sobre a parte de conta tentarei implementar com o MyAnimeList, pois como estou usando um servidor próprio acho que funcionará, mas pelo que você disse deve ser um pouco difícil de integrar, visto que nenhum outro app fez isso.

Sobre a parte de descrição, realmente não há o que eu possa fazer. Eu tenho somente o aplicativo em mãos, os dados que aparecem nas telas vem de sites e API's externas, o que eu não tenho controle nenhum.

**Comment by ZaryImortal** [comment]
Vi tá a um mês sem mexer no código, tô na espera das atualizações. Desiste não cara.

**Comment by LaksCastro** [comment]
Sim, sim, infelizmente não tenho pegado muito no código, como é somente eu de desenvolvedor, agora que estou tendo que desenvolver meu Tcc da escola e trabalhar, fica bem difícil de manter atualizações recorrentes. Mas pode deixar! Todo final de semana agora vou tentar implementar pelo menos um dos comentários das issues! Afinal, eu estava tentando encontrar uma forma de conseguir implementar infinitas API's, sem restrição, ao invés de pegar somente da API do AnimeTV. Assim eu conseguiria oferecer imagens, vídeos, etc. Mas obrigado pela mensagem! Me motivou bastante, coisa que estava faltando, jkkk.

Uma pergunta, na parte de downloads, estou pensando em remover esses 3 apk's leves, e colocar somente 1 que ocupe mais espaço, o que acha @ZaryImortal

**Comment by ZaryImortal** [comment]
Eu não uso a função de baixar, então não sei qual seria melhor. Recomendo priorizar qualidade. Se ele que pesa mais for melhor que os outros 3 então fica com ele.

**Comment by GABIlopez** [comment]
AnimeTV
